packages/app/ui/modal/ModalHeader.tsx

```diff
@@ -3,12 +3,13 @@
 import { Dialog } from "@headlessui/react";
 import { PropsWithChildren } from "react";
 
-export function ModalHeader({ children }: PropsWithChildren) {
+interface ModalHeaderProps extends PropsWithChildren {
+  className?: string;
+}
+
+export function ModalHeader({ children, className }: ModalHeaderProps) {
   return (
-    <Dialog.Title
-      as="div"
-      className="flex items-center w-full px-4 py-3 border-b border-surface-50"
-    >
+    <Dialog.Title as="div" className={className}>
```

Comment:
We probably want to have the same ModalHeader in most modals.

The idea here is to have consistent headers, so passing className has to be a clear edge case. 

_Some styles might be a little different from design because they need some tweaks, they look good on figma but then when coding feel a bit off._

Curious, why did you feel the need to pass the whole classname?

Reply:
Mostly to keep it easy to adapt, another way may be keeping these classes in the `ModalHeader` component and be able to `twMerge` them with the new `className` prop, some sort of default styles (and keep clean all the other files implementing this component)

But was mostly to me tweaking a bit of the existing styles and not wanting those changes to potentially impact other components

Reply:
Agree, let's have the same `ModalHeader` for now as it's much more manageable. We can improve it if we feel the need in the future.

We can keep the `px-4` spacing on the whole modal (instead of the `24px`)

Reply:
Will do!